### PR TITLE
Prepare coverstore/warc.py for Python 3

### PR DESCRIPTION
**Pull request by cclauss**
Some utility functions were removed from urllib in Python 3.

Closes #

### CODE REVIEW

**Comment by hornc** [comment]
I don't know much about coverstore, and am wary of making changes to it lightly. I don't think we have great unit testing around it, and I'm not sure how to test it well locally. Does anyone have experience testing coverstore?

**Comment by tfmorris** [comment]
@hornc Does this not have unit test coverage? Would the solution to your concerns be to write a small unit test? On the surface, the change seems very straightforward.

**Comment by cclauss** [comment]
1. openlibrary/coverstore/tests/test_code.py
2. openlibrary/coverstore/tests/test_coverstore.py
3. openlibrary/coverstore/tests/test_doctests.py
4. openlibrary/coverstore/tests/test_webapp.py

File: openlibrary/coverstore/coverlib.py [hunk 1]
```diff
@@ -3,12 +3,12 @@
 
 import datetime
 import os
+from io import BytesIO
```
Comment (cclauss) [inline openlibrary/coverstore/coverlib.py]: Pillow >= v7.0.0 will assert on StringIO.  See https://github.com/python-pillow/Pillow/pull/4302

Comment (cclauss) [reply openlibrary/coverstore/coverlib.py]: https://travis-ci.org/github/internetarchive/openlibrary/jobs/663471390#L4291 ;-)

**Comment by tfmorris** [comment]
What happened to the branch/PR? Did the fixes move somewhere else?

**Comment by cclauss** [comment]
I am keeping this one on my Bookmarks along with #3043 and will come back to them when things are a bit clearer.  The complicated PRs don't really land so I need to break them into more byte-sized chunks for reviewability.  I had a good pair programming session yesterday with @cdrini so hopefully someday soon I will be able to steer clear of merge commits.

**Pull request closed without merging**